server/tests/lib/nl/date_parser_test.py

```diff
@@ -25,8 +25,8 @@
 class TestDateParser(unittest.TestCase):
 
   @parameterized.expand([
-      ('USA population in May, 2021', [Date('in', 2021, 5)]),
-      ('USA population in Sep 2021', [Date('in', 2021, 9)]),
+      ('USA population in may, 2021', [Date('in', 2021, 5)]),
```

Comment:
To confirm, this is fine because somewhere upstream the small casing happens?

Reply:
yes, I believe so, but thinking more, I think it's better to just have both cases